resources/containers/field-modal.tsx

```diff
@@ -19,35 +19,37 @@ const FieldModal: React.FC<FieldModalProps> = ({
 }) => {
   const [selectedOptions, setSelectedOptions] = useState<string[]>([]);
   const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const [newOption, setNewOption] = useState<string>("");
 
-  // Convert enum to array of values and omit the fieldType
-  const options = Object.values(TYPE_OPTIONS).filter(
+  const typeOptions = Object.values(TYPE_OPTIONS).filter(
     (option) => option !== fieldType,
   );
 
-  useEffect(() => {
-    setSelectedOptions([]);
-  }, [isOpen]);
+  const handleSave = () => {
+    if (fieldType === TYPE_OPTIONS.NUMBER && selectedOptions.length > 0) {
+      onSave({ maxValue: parseInt(selectedOptions[0]) });
```

Comment:
Saving 0 as max value can be dangeorus, is ok if is undefine

Reply:
Due to type constraints is expecting string not string[]

Reply:
Please suggest alternatives